js/script.js

```diff
@@ -1,76 +1,68 @@
 const toDoListHTML = document.querySelector('.toDoList__list');
-const addForm = document.querySelector('form.add');
-const addInput = addForm.querySelector('.adding__input');
+const formElement = document.querySelector('form.add');
+const inputElement = formElement.querySelector('.adding__input');
 
-const getFilms = () => {
-  try {
-    return JSON.parse(localStorage.getItem('movies'));
-  } catch (error) {
-    setFilms([]);
-    return [];
-  }
-};
+const getTodoList = () => JSON.parse(localStorage.getItem('todoList'));
 
-const setFilms = (movies) => localStorage.setItem('movies', JSON.stringify(movies));
+const setTodoList = (todoList) => localStorage.setItem('todoList', JSON.stringify(todoList));
 
-function checkLocalStorage() {
-  const movies = getFilms();
+firstCheckOfTodoList();
+createTodoList();
 
-  if (!movies && !Array.isArray(movies)) {
-    setFilms([]);
-  }
-}
-checkLocalStorage();
-
-createMovieList();
-
-addForm.addEventListener('submit', (event) => {
+formElement.addEventListener('submit', (event) => {
   event.preventDefault();
 
-  let newMovie = addInput.value;
-
-  if (newMovie) {
-    const movies = getFilms();
+  let newTodo = inputElement.value;
 
-    if (newMovie.length > 21) {
-      newMovie = `${newMovie.substring(0, 22)}...`;
+  if (newTodo) {
+    const todoList = getTodoList();
+    if ((!todoList && !Array.isArray(todoList)) || !Array.isArray(getTodoList())) {
+      setTodoList([]);
+    } else {
+      todoList.push(newTodo);
+      setTodoList(todoList);
+      createTodoList();
     }
-
-    movies.push(newMovie);
-    setFilms(movies);
-    createMovieList();
   }
   event.target.reset();
 });
 
-function createMovieList() {
-  addNewString();
-  deleteOneString();
+function firstCheckOfTodoList() {
+  const movies = getTodoList();
+
+  if (!movies && !Array.isArray(movies)) {
+    setTodoList([]);
+  }
+}

```

Comment:
use here `try {} catch() {}` please
rename movies on todoList
Make sure that every element of` todoList` is a string